AutomatedTesting/ShaderLib/scenesrg.srgi

```diff
@@ -14,15 +14,7 @@
 
 #include <Atom/Features/SrgSemantics.azsli>
 
-#define AZ_COLLECTING_PARTIAL_SRG_INCLUDES
-#include <Atom/Feature/Common/Assets/ShaderResourceGroups/ViewSrgIncludesAll.azsli>
-#undef AZ_COLLECTING_PARTIAL_SRG_INCLUDES
-
 partial ShaderResourceGroup SceneSrg : SRG_PerScene
 {
 /* Intentionally Empty. Helps define the SrgSemantic for SceneSrg once.*/
```

Comment:
We should update this comment to say 
/* Intentionally Empty. Add data here based on project's needs */